src/L.Control.Locate.scss

```diff
@@ -15,6 +15,15 @@
   }
 }
 
+.leaflet-touch .leaflet-bar .leaflet-locate-text-active  {
```

Comment:
What does this have to do with the proposed change?

Reply:
This allows to modify the css only if the text option is used, it will not impact the button if the option is not used.

Modifications have been made on the css if the text option is displayed because of the default properties of leaflet which made it impossible to display the text correctly.